src/main/java/Duke.java

```diff
@@ -6,5 +12,49 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________\n" +
+                " Hello! I'm Duke\n" +
+                " What can I do for you?\n" +
+                "____________________________________________________________\n");
+        String line;
+        List<Task> tasks = new ArrayList<>();
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+        while(!line.equals("bye")) {
+            System.out.println("____________________________________________________________\n");
+            if (line.equals("list")) {
```

Comment:
It would be better to use switch-case instead of if-else since there are multiple cases